app/angular/src/client/preview/angular/decorators.test.ts

```diff
@@ -0,0 +1,42 @@
+import { moduleMetadata } from './decorators';
```

Comment:
I think we also need to support an arrays concatenation... 

for example:
```ts
const result = moduleMetadata({
  imports: [FooModule],
})(() => ({
  component: MockComponent,
  moduleMetadata: {
    imports: [BarModule],
  }
}));
```

I would expect to have both a `FooModule` and `BarModule`

Reply:
I have modified it to combine rather than override.